views/error404.html

```diff
@@ -0,0 +1,17 @@
+% include('base.html', title="PyJudge")
```

Comment:
If you want, you can change this so that it properly uses the template system. It's reversed right now. What needs to happen is that this template has to be re-based on top of base.